core/domain/activity_services.py

```diff
@@ -25,10 +25,18 @@
 from core.domain import activity_domain
 from core.platform import models
 
+from typing import List, Tuple
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import activity_models
+
 (activity_models,) = models.Registry.import_models([models.NAMES.activity])
 
+ActivityReferenceList = List[activity_domain.ActivityReference]
+
```

Comment:
Thinking about this I don't think we need this type variable the type is simple enough already.

Reply:
Done!